src/application/components/ClientNotFoundModal.tsx

```diff
@@ -31,19 +32,33 @@ export function ClientNotFoundModal({
     <Modal open={open} onClose={onClose} size="xl">
       <Modal.Header>
         <Modal.Title>Could not find client</Modal.Title>
-      </Modal.Header>
-      <Modal.Body>
-        <p>
+        <Modal.Description>
           An Apollo Client instance was not found, either because an Apollo
           Client instance was never created, or because it could not be
           discovered. This is most commonly fixed by setting the{" "}
           <ConnectToDevToolsOptionLink /> to <code>true</code>.
-        </p>
-        <p className="mt-4">
+        </Modal.Description>
+        <Modal.Description>
           If this was reached in error, please try one of the following
           suggestions or click the &quot;Retry&quot; button to try looking for
           the client again.
-        </p>
+        </Modal.Description>
+        <Modal.Description>
+          Please{" "}
+          <GitHubIssueLink
+            labels={[LABELS.bug]}
+            body={`
+${SECTIONS.defaultDescription}
+${SECTIONS.apolloClientVersion}
+${SECTIONS.devtoolsVersion}
+`}
+          >
+            create an issue
+          </GitHubIssueLink>{" "}
+          if you believe you have reached this in error.
```

Comment:
Specify what "this" is. Use the same term, whether "issue," "error," or something else, as the first suggestion.

```suggestion
          if you believe you received this issue by mistake.
```

Reply:
Thanks for the help! Added a bit more verbosity here to really make it clear. [f841d58](https://github.com/apollographql/apollo-client-devtools/pull/1426/commits/f841d58fadedd657bd4b80cb9b34edb3f361c5dd)